src/PlaywrightSharp.Chromium/Helpers/JsonExtensions.cs

```diff
@@ -0,0 +1,22 @@
+using System;
+using System.IO;
+using System.Text.Json;
+
+namespace PlaywrightSharp.Chromium.Helpers
+{
+    internal static partial class JsonExtensions
+    {
+        public static T ToObject<T>(this JsonElement element, JsonSerializerOptions options = null)

```

Comment:
I'm not sure this is chromium specific

Reply:
Yeah, I moved the class where because the other extension uses the `DefaultChromiumJsonSerializerOptions` but we can have both classes.